MODULE.bazel

```diff
@@ -171,6 +171,7 @@ COMPILERS = {
             name = "{version}_llvm_toolchain".format(version = major_version),
             compile_flags = COMPILE_FLAGS,
             cxx_standard = {"": CXX_STANDARD},
+            extra_compiler_files = "@@toolchains_llvm++llvm+{version}_llvm_toolchain_llvm//:all_includes".format(version = major_version),
```

Comment:
Hrm this smells fishy. Why do we need it?

Reply:
ah yeah, not anymore after wrapping `:all_includes` in a library. this just forces an extra filegroup into the workspace.

Reply:
ggone